apps/pwa/src/pages/my-events.astro

```diff
@@ -0,0 +1,411 @@
+---
+export const prerender = false;
```

Comment:
We're utilising api calls to `/api/my-events`, having this set to `prerender` = `false` doesn't really make this client api call strategy useful.